32blit-sdl/System.cpp

```diff
@@ -269,6 +269,7 @@ void System::loop()
 	blit::joystick.y = shadow_joystick[1];
 	SDL_UnlockMutex(m_input);
 	blit::tick(::now());
+    blit_multiplayer->update();
```

Comment:
I know the indentation is a horrible mess, but that's definitely a bit wonky. (I guess github is getting 2 spaces from the editorconfig file)